AsyncDisplayKit/Layout/ASLayout.h

```diff
@@ -145,6 +138,14 @@ extern BOOL CGPointIsNull(CGPoint point);
 
 @end
 
+@interface ASLayout (Unavailable)
+
+- (instancetype)init __unavailable;
```

Comment:
Question: Never seen `__unavailable` I assume this is the same as `NS_UNAVAILABLE`?